nextjs-starter/baml_src/generators.baml

```diff
@@ -1,6 +1,5 @@
 generator lang_typescript {
-  output_type "typescript"
+  output_type "typescript/react"
```

Comment:
is this react or nextjs btw? i know we chatted about this, but i don't remember if we have any nextjs specific concepts

Reply:
It's all react specific, there isn't anything next.js specific in there. We have another package coming in for the next.js config called `@boundaryml/baml-nextjs-plugin` that is next specific. 

In the docs I've been referencing it as React/NextJS so it's clear for both parties